rls/src/test/java/io/grpc/rls/RlsLoadBalancerTest.java

```diff
@@ -185,6 +193,15 @@ public CachingRlsLbClient.Builder get() {
     };
 
     searchSubchannelArgs = newPickSubchannelArgs(fakeSearchMethod);
+
+    when(mockMetricRecorder.registerBatchCallback(isA(BatchCallback.class),
+        argThat((LongGaugeMetricInstrument metricInstruments) -> {
+          return metricInstruments.getName().equals("grpc.lb.rls.cache_entries");
```

Comment:
Why is this so precise? I'd expect:
```java
when(mockMetricRecorder.registerBatchCallback(any(), any())).thenReturn(() -> {});
```

Or even better:
```java
private MetricRecorder mockMetricRecorder = mock(MetricRecorder.class, delegatesTo(new MetricRecorder() {}));
```

Reply:
Yeah, that was too much. Had to use a when though because I need to return a mock Registration from this call.